WordPress/Classes/Utility/Analytics/WPAppAnalytics.m

```diff
@@ -253,38 +117,14 @@ + (NSError * _Nonnull)sanitizedErrorFromError:(NSError * _Nonnull)error
     return error;
 }
 
-#pragma mark - Usage tracking
-
-+ (BOOL)isTrackingUsage
-{
-    return [[UserPersistentStoreFactory userDefaultsInstance] boolForKey:WPAppAnalyticsDefaultsKeyUsageTracking_deprecated];
-}
-
-- (void)setTrackingUsage:(BOOL)trackingUsage
-{
-    if (trackingUsage != [WPAppAnalytics isTrackingUsage]) {
-        [[UserPersistentStoreFactory userDefaultsInstance] setBool:trackingUsage
-                                                forKey:WPAppAnalyticsDefaultsKeyUsageTracking_deprecated];
-    }
-}
-
 #pragma mark - Tracks Opt Out
 
 - (void)initializeOptOutTracking {
     if ([WPAppAnalytics userHasOptedOutIsSet]) {
         // We've already configured the opt out setting
         return;
     }
-
-    if ([[UserPersistentStoreFactory userDefaultsInstance] objectForKey:WPAppAnalyticsDefaultsKeyUsageTracking_deprecated] == nil) {
-        [self setUserHasOptedOutValue:NO];
-    } else if ([[UserPersistentStoreFactory userDefaultsInstance] boolForKey:WPAppAnalyticsDefaultsKeyUsageTracking_deprecated] == NO) {
-        // If the user has already explicitly disabled tracking,
-        // then we should mirror that to the new setting
-        [self setUserHasOptedOutValue:YES];
-    } else {
-        [self setUserHasOptedOutValue:NO];
-    }
+    [self setUserHasOptedOutValue:NO];
```

Comment:
Is it intentional to delete the code above that reads user settings?

Reply:
I removed the migration code that I assume we no longer need. This runs once.